smart_simulation/Notebooks/modeling.ipynb

```diff
@@ -0,0 +1,942 @@
+{
```

Comment:
<p>I found this book useful: https://otexts.com/fpp2/</p><p> </p><p>It would be good to explicitly that new bag arrivals are a seasonal component.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbpauly/smart-simulation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Noted. I should probably add a brief section explaining those thoughts before exploring the weight data.</p>